en/docs/design/api-policies/regular-gateway-policies/adding-default-endpoints.md

```diff
@@ -1,20 +1,20 @@
-# Adding Dynamic Endpoints
+# Adding Default Endpoints
 
-You cannot dynamically construct the back-end endpoint of an API using the address endpoints in the WSO2 API Manager. To achieve the requirement of a dynamic endpoint, you can use the default endpoint instead. 
+You cannot default construct the back-end endpoint of an API using the address endpoints in the WSO2 API Manager. To achieve the requirement of a default endpoint, you can use the default endpoint instead. 
```

Comment:
"To achieve the requirement of a default endpoint, you can use the default endpoint instead. " This does not make sense ?